tests/cecil-tests/BlittablePInvokes.cs

```diff
@@ -308,9 +310,24 @@ bool IsBlittableTypesWeLike (TypeReference t)
 			return typesWeLike.Contains (t.ToString ());
 		}
 
-		bool IsBlittablePointer (TypeReference type)
+		bool IsBlittablePointer (AssemblyDefinition assembly, TypeReference type, IMarshalInfoProvider provider, StringBuilder result, Dictionary<string, BlitAndReason> blitCache)
 		{
-			return type.IsPointer || type.IsFunctionPointer;
+			if (type.IsFunctionPointer)
+				return true;
+
+			if (!type.IsPointer)
+				return false;
+
+			var localResult = new StringBuilder ();
+			if (!IsTypeBlittable (assembly, type.GetElementType (), provider, result, blitCache)) {
+				var pointerResult = new StringBuilder ();
+				pointerResult.Append ($" {type.Name}: {localResult}");
+				result.Append (pointerResult);
+				blitCache [type.Name] = new BlitAndReason (false, pointerResult.ToString ());
+				return false;
+			}
+
+			return true;
```

Comment:
What is the point of the localResult variable? I don't see it getting any value appended to it at all, nor it is passed to any function within the function.